Addnewplugin.md

```diff
@@ -0,0 +1,31 @@
+# New Plugin in Google-cloud
```

Comment:
Please don't mention google-cloud here, documentation should be generic.

Reply:
Done.